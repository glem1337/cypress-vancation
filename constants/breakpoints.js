// eslint-disable-next-line import/prefer-default-export
export const BREAK_POINTS = {
  MOBILE: 320,
  TABLET: 768,
  WEB: 992,
};
