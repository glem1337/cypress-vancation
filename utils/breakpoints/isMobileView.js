const isMobileView = () => {
  let isMobile = false;

  try {
    const mobileDetector = document.querySelector('#mobile-detector');

    const display = getComputedStyle(mobileDetector)?.display;

    isMobile = display === 'flex';
  } catch (err) {
    isMobile = false;
  }

  return isMobile;
};

export default isMobileView;
