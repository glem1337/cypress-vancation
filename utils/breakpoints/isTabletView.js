const isTabletView = () => {
  let isTablet = false;

  try {
    const mobileDetector = document.querySelector('#tablet-detector');

    const display = getComputedStyle(mobileDetector)?.display;

    isTablet = display === 'flex';
  } catch (err) {
    isTablet = false;
  }

  return isTablet;
};

export default isTabletView;
