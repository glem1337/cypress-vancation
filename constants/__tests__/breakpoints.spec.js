import * as breakpoints from '../breakpoints';

it('breakpoints constants matches snapshot', () => {
  expect(breakpoints).toMatchSnapshot();
});
