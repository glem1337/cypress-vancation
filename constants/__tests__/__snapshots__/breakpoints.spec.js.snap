// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`breakpoints constants matches snapshot 1`] = `
Object {
  "BREAK_POINTS": Object {
    "MOBILE": 320,
    "TABLET": 768,
    "WEB": 992,
  },
}
`;
